atomic_reactor/schemas/plugins.json

```diff
@@ -0,0 +1,38 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+
+  "title": "atomic-reactor plugins configuration",
+
+  "definitions": {
+    "base_plugins_phase": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "name": {"type": "string"},
+          "args": {"type": "object"}
+        },
+        "required": ["name"]
+      }
+    },
+    "general_plugins_phase": {
+      "allOf": [
+        {"$ref": "#/definitions/base_plugins_phase"},
+        {
+          "properties": {
+            "is_allowed_to_fail": {"type": "boolean"},
```

Comment:
Why is this not a property peer to `"name": {"type": "string"}` and `"args": {"type": "object"}` ?

Reply:
I'm pretty sure it is; that's just how you write a schema that expands a ref.

Reply:
Oh I see, so you're expanding because `buildstep_plugins` do not allow these properties. Makes sense.